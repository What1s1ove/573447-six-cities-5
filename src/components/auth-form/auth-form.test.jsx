import * as React from 'react';
import {render} from 'enzyme';
import AuthForm from './auth-form';

describe(`AuthForm component`, () => {
  it(`should render AuthForm component`, () => {
    const component = render(
        <AuthForm
          formState={{}}
          onFormSubmit={jest.fn()}
          onFormStateChange={jest.fn()}
        />
    );

    expect(component).toMatchSnapshot();
  });
});
