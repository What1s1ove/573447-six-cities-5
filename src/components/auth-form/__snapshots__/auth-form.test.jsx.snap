// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthForm component should render AuthForm component 1`] = `
<form
  action="#"
  class="login__form form"
  method="post"
>
  <div
    class="login__input-wrapper form__input-wrapper"
  >
    <label
      class="visually-hidden"
    >
      E-mail
    </label>
    <input
      class="login__input form__input"
      name="email"
      placeholder="Email"
      required=""
      type="email"
      value=""
    />
  </div>
  <div
    class="login__input-wrapper form__input-wrapper"
  >
    <label
      class="visually-hidden"
    >
      Password
    </label>
    <input
      class="login__input form__input"
      name="password"
      placeholder="Password"
      required=""
      type="password"
      value=""
    />
  </div>
  <button
    class="login__submit form__submit button"
    type="submit"
  >
    Sign in
  </button>
</form>
`;
